### PR TITLE
Update `People/Users_with_unique_titles` to include MWC 4K 2021 winners

### PR DESCRIPTION
**Pull request by Nivalyx**
also added one FA ([Redside](http://osu.ppy.sh/users/16039046)) and a couple of Elite Mapper names ([Sylas](http://osu.ppy.sh/users/3906405)/[kuyusu](http://osu.ppy.sh/users/11758667)) that haven't been included in the list yet into the wiki page.

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/en.md [hunk 1]
```diff
@@ -352,6 +357,7 @@ These players earned their titles through either a crazy gameplay achievement or
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/en.md:358]:
```suggestion

```
His title was removed after the whole tournament ordeal a couple of months back.

File: wiki/People/Users_with_unique_titles/fr.md [hunk 2]
```diff
@@ -346,6 +351,7 @@ Ces joueurs ont obtenu leur titre soit en réalisant un exploit de jeu fou, soit
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/fr.md:352]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/id.md [hunk 3]
```diff
@@ -346,6 +351,7 @@ Pemain-pemain berikut memperoleh gelarnya masing-masing atas keberhasilan mereka
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/id.md:352]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/pl.md [hunk 4]
```diff
@@ -345,6 +350,7 @@ Poniżsi gracze otrzymali swoje tytuły za znaczące osiągnięcia w grze lub uk
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/pl.md:351]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/pt-br.md [hunk 5]
```diff
@@ -350,6 +355,7 @@ Esses jogadores conseguiram seus títulos com alguma conquista maluca ou complet
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/pt-br.md:356]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/th.md [hunk 6]
```diff
@@ -354,6 +359,7 @@ Storyboarders ที่แสดงความสามารถในการ
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/th.md:360]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/tr.md [hunk 7]
```diff
@@ -350,6 +355,7 @@ Bu oyuncular ya oyunda çılgın bir başarım elde ederek ya da belirli bir zor
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/tr.md:356]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/zh.md [hunk 8]
```diff
@@ -346,6 +351,7 @@ osu!团队为社区成员举办了艺术比赛，使他们有机会设计或重
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```
Comment (LeoFLT) [inline wiki/People/Users_with_unique_titles/zh.md:352]:
```suggestion

```

File: wiki/People/Users_with_unique_titles/en.md (hunk 1 above)
Comment (Nivalyx) [reply wiki/People/Users_with_unique_titles/en.md:358]: yeah

**Review by TicClick — APPROVED** [review]

**Pull request merged**